sympy/printing/str.py

```diff
@@ -829,6 +829,22 @@ def _print_Quaternion(self, expr):
         a = [s[0]] + [i+"*"+j for i, j in zip(s[1:], "ijk")]
         return " + ".join(a)
 
+    def _print_Dual(self, expr):
+        from sympy.algebras import Quaternion, Dual
+        s = [self.parenthesize(i, PRECEDENCE["Mul"], strict=True) for i in expr.args]
+        a = []
+        if isinstance(expr.args[0], (Quaternion, Dual)):
+            a.append("(" + s[0] + ")")
+        else:
+            a.append(s[0])
+
+        if isinstance(expr.args[1], (Quaternion, Dual)):
+            a.append("("+s[1]+")*ð")
+        else:
+            a.append(s[1]+"*ð")
```

Comment:
I think this should avoiding using `ð` unless unicode printing has been opted into. From what I remember, the default is ascii.

Reply:
Yeah I hoped that since ð is extended ascii it might still work, but the style checker bounces this as well. So now I'm unsure which symbol to use.